src/routes/notification.route.js

```diff
@@ -10,10 +11,11 @@ const { checkNotificationExists, checkNotificationOwner } = NotificationMiddlewa
 const {
   getUserNotifications, updateNotificationStatus, getSingleNotification, markAllNotificationsAsRead
 } = NotificationController;
+const { validateNotificationStatus } = Validation;
 
 app.get('/', checkToken, getUserNotifications);
 app.get('/:id', checkToken, checkNotificationExists, checkNotificationOwner, getSingleNotification);
-app.patch('/:id/:status', checkToken, checkNotificationExists, checkNotificationOwner, updateNotificationStatus);
+app.patch('/:id/:status', validateNotificationStatus, checkToken, checkNotificationExists, checkNotificationOwner, updateNotificationStatus);
```

Comment:
Use the approach of a toggle/switch. You don't need the user to pass in data to read or unread. Just one route to handle both. If `isRead` is `true`, change to `false`. If `false`, change to `true`. Check out `/readall` for reference

Reply:
great